code/modules/mapping/map_template.dm

```diff
@@ -136,7 +140,12 @@
 	// ruins clogging up memory for the whole round.
 	var/datum/parsed_map/parsed = cached_map || new(file(mappath))
 	cached_map = keep_cached_map ? parsed : null
-	if(!parsed.load(T.x, T.y, T.z, cropMap=TRUE, no_changeturf=(SSatoms.initialized == INITIALIZATION_INSSATOMS), placeOnTop=TRUE))
+	var/list/turf_blacklist = list()
+	turf_blacklist += update_blacklist(T)
```

Comment:
If this proc returns `null`, this will populate the list with nulls. Is that intended?

Reply:
no i didnt know that was a thing, now turf_blacklist is an argument for update_blacklist() and gets added to if update_blacklist() has been overridden, otherwise nothing happens to it and no nulls are added